src/valhalla_run_route.cc

```diff
@@ -154,8 +154,7 @@ const valhalla::TripLeg* PathTest(GraphReader& reader,
       LOG_INFO("Try again with relaxed hierarchy limits");
       cost->set_pass(1);
       pathalgorithm->Clear();
-      const float expansion_within_factor = (using_astar) ? 4.0f : 2.0f;
```

Comment:
keeping this binary up to date with the server has been a constant point of failure for the project. i have always advocated for stripping this binary down to just calling the same bits of code that the server does rather than having its own slightly different implementation.

i am certain that, with enough care, we can get there but in the mean time thank you for cleaning this up

Reply:
Well, with the advent of `valhalla_service.cc`, we could simply remove the `valhalla_run_route.cc` completely. Unless there are strong reasons to not to.

Reply:
that is what i would like to do too but... there are currently some uses of `valhalla_run_route` in the scripting that we use to do regression testing. if we can port the formatting and functionality out of this binary to satisfy those use cases or change those downstream processes to use the other tools then we could do this, but today is sadly not yet that day :smile:

Reply:
I see. I wish I had more time, but this year is hectic here too :-)